src/main/java/io/pravega/connectors/flink/AbstractStreamingWriterBuilder.java

```diff
@@ -23,9 +23,9 @@
  */
 public abstract class AbstractStreamingWriterBuilder<T, B extends AbstractStreamingWriterBuilder> extends AbstractWriterBuilder<B> {
 
-    // the numbers below are picked somewhat arbitrarily at this point
-    private static final long DEFAULT_TXN_TIMEOUT_MILLIS = 2 * 60 * 60 * 1000; // 2 hours
-    private static final long DEFAULT_TX_SCALE_GRACE_MILLIS = 10 * 60 * 1000; // 10 minutes
+    // the numbers below are picked based on the default max settings in Pravega
+    private static final long DEFAULT_TXN_TIMEOUT_MILLIS = 30000; // 30 seconds
+    private static final long DEFAULT_TX_SCALE_GRACE_MILLIS = 30000; // 30 seconds
```

Comment:
The scale grace period is a parameter that is supposed to go away with the rolling txns feature. Currently, it cannot be smaller than the lease period.